generator-elide/generators/app/model.js

```diff
@@ -0,0 +1,94 @@
+const generator = require('./generator');
+
+const srcJavaPath = 'src/main/java';
+
+const choices = [
+  'String',
+  'Int',
+  'Short',
+  'Float',
+  'Double',
+  'Long',
+  'Long long',
+  'Boolean',
+  'Char'
+];
+
+const models = [];
+
+let newModelAttributes = {
+  name: '',
+  fields: []
+};
+
+const fieldPrompt = (yo, pomObj) => {
+	return yo.prompt([{
+		type: 'input',
+		name: 'name',
+		message: 'Field name?'
+	}, {
+		name: 'type',
+		message: 'What type?',
+		type: 'list',
+		choices
+	}]).then((model) => {
+        if (model.type != "String") {
+            model.type = model.type.toLowerCase();
+        }
+        const { name, type } = model;
```

Comment:
Since it doesn't seem that model is used anywhere explicitly in this context, can't you unpack this as the argument? (i.e. `.then(({name, type}) => ...)`)

This is assuming `model.type = ...` isn't mutating `model` for another context. If it is, we should revisit why that's the case so we don't get bitten by it in the future.